_container_scripts/keyper-db-init.sh

```diff
@@ -2,4 +2,15 @@
 
 set -e
 
-createdb -U postgres keyper
+echo "Checking for backup dump file..."
+if [ -f "/var/lib/postgresql/dump/keyper.dump" ]; then
+    echo "Backup dump found, restoring database with full schema and data..."
+    # Create the database first
+    createdb -U postgres keyper 2>/dev/null || echo "Database already exists"
```

Comment:
This is not necessary. You're already using `--create` below and additionally these init scripts are only called if no database files exist.